src/types/task.ts

```diff
@@ -90,4 +90,6 @@ export enum TaskStatus {
   // not satisfied
   StatusBlocked = "blocked",
   Aborted = "aborted",
+
```

Comment:
extra space

Reply:
I put the space there on purpose because it's not part of the above group and comment.